lib/msal-common/src/authority/Authority.ts

```diff
@@ -438,33 +443,46 @@ export class Authority {
      * @param newMetadata
      */
     private async updateCloudDiscoveryMetadata(metadataEntity: AuthorityMetadataEntity): Promise<AuthorityMetadataSource> {
+        this.logger.verbose("Attempting to get cloud discovery metadata in the config");
         let metadata = this.getCloudDiscoveryMetadataFromConfig();
         if (metadata) {
+            this.logger.verbose(`Found the following cloud discovery metata in the config: ${JSON.stringify(metadata)}`);
             metadataEntity.updateCloudDiscoveryMetadata(metadata, false);
             return AuthorityMetadataSource.CONFIG;
         }
+        this.logger.verbose("Did not find cloud discovery metadata in the config");
 
-        // If The cached metadata came from config but that config was not passed to this instance, we must go to the network
+        // If the cached metadata came from config but that config was not passed to this instance, we must go to the network
+        this.logger.verbose("Attempting to get cloud discovery metadata in the cache");
         if (this.isAuthoritySameType(metadataEntity) && metadataEntity.aliasesFromNetwork && !metadataEntity.isExpired()) {
+            this.logger.verbose(`Found the following metadata in the cache: ${JSON.stringify(metadata)}`);
             // No need to update
             return AuthorityMetadataSource.CACHE;
         }
+        this.logger.verbose("Did not find cloud discovery metadata in the cache");
 
-        const harcodedMetadata = this.getCloudDiscoveryMetadataFromHarcodedValues();
-
+        this.logger.verbose("Attempting to get cloud discovery metadata from the network");
         metadata = await this.getCloudDiscoveryMetadataFromNetwork();
```

Comment:
Is it worth adding some more logging statements inside `getCloudDiscoveryMetadataFromNetwork` ?